### DISCUSSION TITLE
Is this project still supported?

### DISCUSSION THREAD

**Discussion by ghxstweb** [touilleMan/godot-python#400] [General]
I really enjoyed using godot-python back when Godot was still version 3.x! Is this project still being supported? If so, I'd like to know more how I might be able to look into the code and assist with anything

**Comment by GearUnclear** [comment]
I would also like to know

**Comment by touilleMan** [comment]
Hi there,

In a nutshell, Godot 4 broke entirely the C API (switching from GDNative to GDExtension), while this was objectively the right thing to do (the new API is much more powerful), this entirely broke Godot-Python.

So a lot of work is needed to make Godo-Python work on Godot 4. On top of that my personal time has been very limited for the last years (crazy amount of professional work, plus now I have a daughter to care for :smile: ), so progress is slow.

However I'm 100% confident we will eventually get Godot-Python for Godot 4 (I'm especially planning to have less work in my daily job in the future ^^)

**Reply by hemna** [reply]
Ok, so this probably explains why I can't find Python anything in the asset lib of my newly installed godot 4.2

![Screenshot 2024-03-19 at 11 20 51 AM](https://github.com/touilleMan/godot-python/assets/672207/a21415cf-23b2-4146-a6ff-e1fcc050f894)
